src/main/simulation.cpp

```diff
@@ -304,7 +304,7 @@ bool Dissolve::iterate(int nIterations)
 
                 Messenger::heading("{} ({})", ModuleTypes::moduleType(module->type()), module->name());
 
-                if (!module->executeProcessing(*this, worldPool()))
+                if (static_cast<int>(module->executeProcessing(*this, worldPool())) == 0)
```

Comment:
Ick!  Let's not do this as it really hides intent, and kinda undoes all the niceness of `enum class`.  All we need to do is make the enum `public` in the `Module` class.
```suggestion
                if (module->executeProcessing(*this, worldPool()) == Module::ExecutionResult::Failed)
```